build.xml

```diff
@@ -1528,7 +1530,7 @@
         <jvmarg value="-Dcassandra.testtag=${tag}.jdk${ant.java.version}"/>
         <jvmarg value="-Dcassandra.keepBriefBrief=${cassandra.keepBriefBrief}" />
         <jvmarg value="-Dcassandra.strict.runtime.checks=true" />
-        <jvmarg value="-Dcassandra.test.flush_local_schema_changes=${cassandra.test.flush_local_schema_changes}"/>
+        <jvmarg value="-Dcassandra.unsafesystem=${cassandra.unsafesystem}"/>
```

Comment:
Could we make this name less generic? unsafe, but in what way?
Suppose I'd like to use this property in the code. When am I allowed to? When does it make sense?
I'd like this to be commented upon and more self-evident from the property name.
Do you think this can be done?